extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js

```diff
@@ -0,0 +1,77 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusWebAds',[
+	'ext.wikia.adEngine.context.slotsContext',
+	'wikia.geo',
+	'wikia.instantGlobals',
+	'wikia.log'
+], function (slotsContext, geo, instantGlobals, log) {
+	'use strict';
+
+	var bidderName = 'appnexusWebAds',
+		aliases = {
+			'appnexus': [bidderName]
```

Comment:
Did you consider using `appnexusAst` adapter? It's more modern and performs just 1 request for multiple bids.

Reply:
Right. Switched to appnexusAst.